### PR TITLE
Post liking capabilities

### PR DESCRIPTION
**Pull request by michael-szczepanski**
Will need to run `npm install` in the frontend

1. .DS_Store added to .gitignore (I will stop doing this eventually, I promise)

Backend
1. Added a new method to PostController that currently only updates the likes field in the database
2. Added likes field to Post model
3. A patch route added to Post router

Frontend
4. a jwt_decoder package installed and used to retrieve and store userId in Feed from the token
5. Post component recreated from scratch with a function added to update the likes field
6. Slightly updated the html code for the Post component to display the button and number of likes

### CODE REVIEW

**Review by sarahc-dev — APPROVED** [review]
Amazing. Love it!

**Review by ChristieGriffiths — COMMENTED** [review]
Great job!

File: frontend/src/components/post/Post.js
Comment (wimdavies) [inline frontend/src/components/post/Post.js:1]: Double Post issue

**Review by wimdavies — CHANGES_REQUESTED** [review]
Incorporate FE component Post.js  `post` s into one

File: frontend/src/components/post/Post.js
Comment (michael-szczepanski) [reply frontend/src/components/post/Post.js:1]: Hopefully resolved now.

**Review by wimdavies — APPROVED** [review]
Kosher.

**Pull request merged**